US-EPA-CAMD/easey-ecmps-ui
# Fix/5984 save group id

## Main Changes

- Fixed improper casing in group ID accessors throughout the application (`groupID` -> `groupId`). This fixed a similar issue in the _Monitoring Plans_ module, too.

## Steps to test

1. Test the steps outlined in this issue comment: https://github.com/US-EPA-CAMD/easey-ui/issues/5984#issuecomment-2050098027.
2. Open the _Monitoring Plans_ module for any location, and select the **Defaults** section.
3. Edit one of the rows under the **Defaults** section, enter a **Group ID**, and click **Save and Close**.
4. Reopen the same row, and confirm the **Group ID** remains populated.
